--- automake-1.13.1/bin/aclocal.in.0000~	2013-01-01 12:50:12.000000000 +0100
+++ automake-1.13.1/bin/aclocal.in	2013-04-23 13:16:02.405057615 +0200
@@ -81,6 +81,9 @@ my @diff_command;
 # --dry-run
 my $dry_run = 0;
 
+# --dont-fix
+my $dont_fix = 0;
+
 # configure.ac or configure.in.
 my $configure_ac;
 
@@ -1058,6 +1061,7 @@ sub parse_arguments ()
      'system-acdir=s'	=> sub { shift; @system_includes = @_; },
      'automake-acdir=s'	=> sub { shift; @automake_includes = @_; },
      'diff:s'		=> \$diff_command,
+     'dont-fix'		=> \$dont_fix,
      'dry-run'		=> \$dry_run,
      'force'		=> \$force_output,
      'I=s'		=> \@user_includes,
@@ -1129,6 +1133,7 @@ sub parse_ACLOCAL_PATH ()
 
 parse_WARNINGS;		    # Parse the WARNINGS environment variable.
 parse_arguments;
+system("fix-old-automake-files") if(!$dont_fix);
 parse_ACLOCAL_PATH;
 $configure_ac = require_configure_ac;
 
